src/AppInstallerCommonCore/Public/AppInstallerErrors.h

```diff
@@ -100,6 +100,7 @@
 #define APPINSTALLER_CLI_ERROR_PORTABLE_SYMLINK_PATH_IS_DIRECTORY ((HRESULT)0x8A150055)
 #define APPINSTALLER_CLI_ERROR_INSTALLER_PROHIBITS_ELEVATION    ((HRESULT)0x8A150056)
 #define APPINSTALLER_CLI_ERROR_PORTABLE_UNINSTALL_FAILED          ((HRESULT)0x8A150057)
+#define APPINSTALLER_CLI_ERROR_UNSUPPORTED_ARGUMENT             ((HRESULT)0x8A150058)

```

Comment:
Can you also add it to the test constants and add the mapping to string? I think the files are something like `TestConstants.cs` and `Errors.cpp`

Reply:
No longer applicable because I removed the error